CraftCMS.gitignore

```diff
@@ -1,3 +1,4 @@
-# Craft Storage (cache) [http://buildwithcraft.com/help/craft-storage-gitignore]
+# Craft 2 Storage (https://craftcms.com/support/craft-storage-gitignore)
+# not necessary for Craft 3 (https://github.com/craftcms/craft/issues/26)
 /craft/storage/*
-!/craft/storage/logo/*
\ No newline at end of file
+!/craft/storage/rebrand
```

Comment:
This change isn't clear to me - what's the context here?

Reply:
This is the Craft team's recommendation, from https://craftcms.com/support/craft-storage-gitignore